google_photos_takeout_helper/__main__.py

```diff
@@ -226,7 +230,7 @@ def find_duplicates(path: Path, filter_fun=lambda file: True):
                 files_by_size[file_size].append(file)
 
         # For all files with the same file size, get their hash on the first 1024 bytes
-        for file_size, files in files_by_size.items():
+        for file_size, files in tqdm(files_by_size.items()):
```

Comment:
Neither here :confused:

Reply:
This use of tqdm also works on my console.